src/docfx/build/xref/ExternalXrefSpec.cs

```diff
@@ -30,16 +30,16 @@ internal record ExternalXrefSpec : IXrefSpec
 
     FilePath? IXrefSpec.DeclaringFile => null;
 
-    public string? GetXrefPropertyValueAsString(string propertyName)
+    public (string? propertyValue, bool localizable) GetXrefPropertyValueAsString(string propertyName)
```

Comment:
>string? propertyValue, bool localizable [](http://example.com/codeflow?start=12&length=39)

consider invent `Localizable<T>` to encapsulate `(string?, bool)` and `(JToken?, bool)`

Reply:
Done